core/src/main/java/com/amplifyframework/core/DeveloperMenuActivity.java

```diff
@@ -63,6 +74,16 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
         }
     }
 
+    @Override
+    public void onBackPressed() {
```

Comment:
The DeveloperMenuActivity is finished by default (if I don't override onBackPressed) if the user presses the phone's back button, but that means the user can no longer activate the developer menu.

Reply:
Ahh, I just took at look at the changes you made in `Amplify.configure()`.   By design, an [`Activity`](https://developer.android.com/reference/android/app/Activity) should really only be used when there is UI to be displayed, so I don't think it's a good idea to create an `Activity` until the dev menu is actually supposed to be shown.    

Another issue I think you will have with the current approach is this scenario:
 - In the customer's app, they call `Amplify.configure()`, which starts the `DeveloperMenuActivity`
 - Then the customer creates one or more of their own activities for their app. 
 - Then, the customer shakes the phone.  Now, `DeveloperMenuActivity` is not the foreground activity, so even if you change visibility of elements on the dev menu, it won't be shown, since it is behind other activities.

Instead, perhaps you could turn `ShakeDetector` into a singleton, which is initialized in `Amplify.configure`, and is always listening for shake events.  Then, when the ShakeDetector detects a shake, it could start the `DeveloperMenuActivity`?

Reply:
Thanks for pointing this out. I just tested it and you're right that the DeveloperMenuActivity isn't visible even when shaking the phone if another activity is started.